Kickstarter-iOS/AppDelegate.swift

```diff
@@ -3,9 +3,7 @@ import AppCenterAnalytics
 import AppCenterCrashes
```

Comment:
Let's remove `AppCenterAnalytics` and `AppCenterCrashes` as well as their linked frameworks and protocol conformance as we're not using either.

We can probably remove `crashManagerDidFinishSendingCrashReport()` as a VM input too unless we want to implement a similar callback from Crashlytics.